plugins/modules/aci_ip_aging.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_ip_aging
+short_description: Manage IP Aging Control (ep:IpAgingP)
+description:
+- Manage IP Aging Control on Cisco ACI fabrics.
+options:
+  enabled:
```

Comment:
Should we call this admin_state since we use this throughout the repository in various other inputs? What is the UI behaviour?

Reply:
Another option could be to absent state for disabling and present for enabling. What is you thought on this?

Reply:
I've renamed the parameter to admin_state, with enabled as one of the aliases. I think it would be better to avoid using absent state for this, to try and keep behaviour as consistent as possible with the existing modules.